apps/web/modules/availability/[schedule]/schedule-view.tsx

```diff
@@ -76,15 +72,7 @@ export const AvailabilitySettingsWebWrapper = ({
 
   const updateMutation = trpc.viewer.availability.schedule.update.useMutation({
     onSuccess: async ({ prevDefaultId, currentDefaultId, ...data }) => {
-      if (prevDefaultId && currentDefaultId) {
-        // check weather the default schedule has been changed by comparing  previous default schedule id and current default schedule id.
-        if (prevDefaultId !== currentDefaultId) {
-          // if not equal, invalidate previous default schedule id and refetch previous default schedule id.
-          utils.viewer.availability.schedule.get.invalidate({ scheduleId: prevDefaultId });
-          utils.viewer.availability.schedule.get.refetch({ scheduleId: prevDefaultId });
-        }
-      }
```

Comment:
this is simply replaceable by calling `revalidatePage` (since this will invalidate and refetch the updated schedule)